### ISSUE TITLE
SI/PCIe: Every bundle pin of a supported bundle must have a support option.

### ISSUE THREAD

**Issue by Johnsel** [JITx-Inc/jsl#58]
Hi all,

I'm getting the following error when I comment out PRSNT# in my module definition:
`stanza> reload
Uncaught Exception: components\ASM2464PD.stanza:1248.2: Every bundle pin of a supported bundle must have a support option.
  in components/ASM2464PD
    at components\ASM2464PD.stanza:1249.20
  in main
    at c:\\\\Users\\\\jamms\\\\OneDrive\\\\Documenten\\\\- JITX\\\\ASM2464PD_Demo_2\\\\main.stanza:23.7
  in main
    at c:\\\\Users\\\\jamms\\\\OneDrive\\\\Documenten\\\\- JITX\\\\ASM2464PD_Demo_2\\\\main.stanza:46.0`

See relevant part of definition:

```
val bd-4x = pcie(4)
  supports bd-4x :
    for i in 0 to 4 do :
      bd-4x.data.rx[i].P => sw.PRXP[i]
      bd-4x.data.rx[i].N => sw.PRXN[i]
      bd-4x.data.tx[i].P => sw.PTXP[i]
      bd-4x.data.tx[i].N => sw.PTXN[i]
    bd-4x.data.refclk.P => sw.REFCLKP[0]
    bd-4x.data.refclk.N => sw.REFCLKN[0]

    bd-4x.control.PEWAKE# => sw.GPIO[22]
    ;bd-4x.control.PRSNT#  => sw.GPIO[6]  <-- problem here
    bd-4x.control.PERST#  => sw.PERST0#
    bd-4x.control.CLKREQ# => sw.GPIO[8]
```

I would suggest making at least PRSNT# optional, since it is only required for desktop-style PCIe slots, and not for M.2 type connectors.

Best regards,

John

**Comment by bhusang** [comment]
@Johnsel Good information to have. I have a PR in motion that will make the pin optional. https://github.com/JITx-Inc/jsl/pull/59

**Comment by bhusang** [comment]
PR merged, release v0.3.1 available.
